examples/counter/src/App.js

```diff
@@ -85,6 +85,26 @@ const Controls = () => {
     </div>
   )
 }
+
+const ButtonUp = ({ onClick }) => (
+  <svg viewBox="0 0 430 452" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      onClick={onClick}
+      d="M214.83 0.95459C82.4432 0.95459 0.0568237 91.2955 0.0568237 226.523C0.0568237 272.651 9.76549 313.624 27.8727 347.545L340.5 36.3569C306.7 13.5435 264.249 0.95459 214.83 0.95459Z"
+      fill="#A5FFCE"
+    />
+  </svg>
+)
+
+const ButtonDown = ({ onClick }) => (
+  <svg viewBox="0 0 430 452" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      onClick={onClick}
+      d="M214.83 451.523C347.216 451.523 429.602 360.898 429.602 226.523C429.602 187.816 422.852 152.786 410.112 122.5L106 426.214C136.689 442.604 173.299 451.523 214.83 451.523Z"
+      fill="#FFBEC2"
+    />
+  </svg>
+)
```

Comment:
Oh, you want to put them in. Okay, why not.

Reply:
> Oh, you want to put them in. Okay, why not.

I'm sorry, I'm not good at English, can you explain in more detail?

Reply:
It was intentional to avoid having `ButtonUp` and `ButtonDown` there. But, we can have them.

Reply:
I added it using ButtonUp and ButtonDown in lines 83~84.

Is this the problem now?

Reply:
No, it's not a problem. I will merge this.
(As there was an intention not to have it, we may remove it in the future.)